### PR TITLE
docs(get in touch): changes to getting in touch for help and feedback

### PR DESCRIPTION
**Pull request by robertjmccarthy**
### Description of the change

Updating how users of the MOJ Design System can get in touch for general help and feedback. Further updates will come in future changes to include how users can get support on specific components and patterns, as well as give feedback and suggest changes on those specific components and patterns.

### CODE REVIEW

File: docs/get-in-touch.md
Comment (gregtyler) [inline docs/get-in-touch.md:1]: This is linked to from a few files, so you'll need to update the links in them too:
- [CONTRIBUTING.md](https://github.com/ministryofjustice/moj-frontend/blob/722f7e09fbb4a70ef9913b8f9e1e0c08afba9c11/CONTRIBUTING.md?plain=1#L11)
- [CODE_OF_CONDUCT.md](https://github.com/ministryofjustice/moj-frontend/blob/722f7e09fbb4a70ef9913b8f9e1e0c08afba9c11/CODE_OF_CONDUCT.md?plain=1#L37)
- [.github/ISSUE_TEMPLATE/BUG_REPORT.md](https://github.com/ministryofjustice/moj-frontend/blob/722f7e09fbb4a70ef9913b8f9e1e0c08afba9c11/.github/ISSUE_TEMPLATE/BUG_REPORT.md?plain=1#L12)
- [.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md](https://github.com/ministryofjustice/moj-frontend/blob/722f7e09fbb4a70ef9913b8f9e1e0c08afba9c11/.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md?plain=1#L12)

Comment (robertjmccarthy) [reply docs/get-in-touch.md:1]: Thanks Greg. I'll update those in a separate PR.

Comment (robertjmccarthy) [reply docs/get-in-touch.md:1]: Made those changes, thanks for locating them Greg - https://github.com/ministryofjustice/moj-frontend/pull/533/commits/a987e5f03d4354c10af5ba35cd12ccc2130180cf

**Review by gregtyler — APPROVED** [review]

**Pull request merged**